### PR TITLE
kube_service to pgview link

### PR DESCRIPTION
**Pull request by erthalion**
Add deeplink with a reference to pgview for kube_service

### CODE REVIEW

File: zmon_agent/discovery/kubernetes/cluster.py [hunk 1]
```diff
@@ -192,7 +198,7 @@ def get_entities(self, **kwargs) -> list:
 
                     postgresql_cluster_member_entities = get_postgresql_cluster_members(
                         self.kube_client, self.cluster_id, self.alias, self.environment, self.region,
-                        self.infrastructure_account, self.hosted_zone_format_string, namespace=self.namespace)
+                        self.infrastructure_account, self.hosted_zone_format_string, namesace=self.namespace)
```
Comment (Jan-M) [inline zmon_agent/discovery/kubernetes/cluster.py]: where did the "p" go?

File: zmon_agent/discovery/kubernetes/cluster.py [hunk 2]
```diff
@@ -367,11 +378,16 @@ def get_cluster_services(
...
 
             'service_name': service.name,
-            'service_namespace': obj['metadata']['namespace'],
+            'service_namespace': service_namespace,
             'service_type': service_type,
             'service_ports': obj['spec'].get('ports', None),  # Could be useful when multiple ports are exposed.
 
             'endpoints_count': endpoints_map.get(service.name, 0),
+
+            'deeplink1': '{}/#/clusters/{}/{}'.format(
```
Comment (Jan-M) [inline zmon_agent/discovery/kubernetes/cluster.py]: This should only be applied if application=spilo, pgview does not make sense to all services

**Comment by Jan-M** [comment]
Looks good to me now.

**Comment by erthalion** [comment]
:+1:

**Comment by vetinari** [comment]
:+1:

**Review by vetinari — APPROVED** [review]

**Pull request merged**